### PR TITLE
Change: Replaces google search with fandom wiki internal search logic

### PR DESCRIPTION
**Pull request by ErymanthusRayDeeUx**

### CODE REVIEW

**Comment by ErymanthusRayDeeUx** [comment]
restore #485

File: src/main/java/at/hannibal2/skyhanni/features/commands/WikiCommand.kt [hunk 1]
```diff
@@ -26,9 +26,12 @@ class WikiCommand {
                 val search = packet.message.substring(6)
                 LorenzUtils.chat("§e[SkyHanni] Searching the Fandom Wiki for §c$search")
 
-                val url = "https://www.google.com/search?q=inurl%3Ahypixel-skyblock.fandom.com $search&hl=en"
+                //val url = "https://www.google.com/search?q=inurl%3Ahypixel-skyblock.fandom.com $search&hl=en"
+                //val poggers = true
+                //val google = !poggers
+                val url = "https://hypixel-skyblock.fandom.com/wiki/Special:Search?query=$search&scope=internal"
```
Comment (lineargraph) [inline src/main/java/at/hannibal2/skyhanni/features/commands/WikiCommand.kt:32]: Maybe we could remove those comments (and also use a setting for this)

**Review by lineargraph — CHANGES_REQUESTED** [review]

File: src/main/java/at/hannibal2/skyhanni/features/commands/WikiCommand.kt (hunk 1 above)
Comment (RayDeeUx) [reply src/main/java/at/hannibal2/skyhanni/features/commands/WikiCommand.kt:32]: hanni said, and i quote:
image

to be fair it saves a lot of grief on choosing a search engine and instead goes straight into the horse's mouth when it comes to wiki searches

Comment (RayDeeUx) [reply src/main/java/at/hannibal2/skyhanni/features/commands/WikiCommand.kt:32]: also *wow* i need to turn my github notifications on more often i have missed a lot of comments from you

**Review by hannibal002 — CHANGES_REQUESTED** [review]
image

**Comment by hannibal002** [comment]
closing, since https://github.com/hannibal002/SkyHanni/pull/494 exists.

**Pull request closed without merging**